blocks/library/button/index.js

```diff
@@ -74,7 +74,7 @@ registerBlock( 'core/button', {
 			<span className="blocks-button" title={ title }>
 				<Editable
 					tagName="span"
-					placeholder={ wp.i18n.__( 'Write some text…' ) }
+					placeholder={ wp.i18n.__( 'Write label…' ) }
```

Comment:
> This is a sting issue really. :)

"Sting issue"?

I'm sure we'll find a solution, even if we have to do it through clever labels and liberal min-widths.